Releases.md

```diff
@@ -0,0 +1,13 @@
+# Process for Releasing
+
+First before creating a release, make sure that [the version number in `Directory.Build.Props`](https://github.com/justeat/JustSaying/blob/d30543fbfc3cf640835339efbe497466e230f220/Directory.Build.props#L22) matches the major and minor version you are looking to release.
```

Comment:
We couldn't need to do this is we opt to rely on tags only. If we did that then there's no risk of them not being aligned.

Reply:
This property doesn't affect the release process. The release is purely driven by the tag you create. Updating this property is actually something you'd do _after_ releasing, in the case where you want to start work on a new major-minor range.

Reply:
Thanks, I'll update this 👍